tests/php/sync/test_class.jetpack-sync-comments.php

```diff
@@ -93,6 +93,62 @@ public function test_wp_untrash_comment() {
 		$this->assertEquals( 0, $this->server_replica_storage->comment_count( 'trash' ) );
 	}
 
+	public function test_sends_highlander_comment_meta_with_comment() {
```

Comment:
This test should be removed. We no longer look at this "meta" attribute on the server.